packages/react-components/react-switch/src/components/Switch/Switch.types.ts

```diff
@@ -37,7 +37,10 @@ export type SwitchOnChangeData = {
 /**
  * Switch Props
  */
-export type SwitchProps = Omit<ComponentProps<Partial<SwitchSlots>, 'input'>, 'onChange'> & {
+export type SwitchProps = Omit<
+  ComponentProps<Partial<SwitchSlots>, 'input'>,
+  'checked' | 'defaultChecked' | 'onChange'
+> & {
```

Comment:
As we have own definition we need to exclude existing ones for `checked` & `defaultChecked`. It also solves problem with the description for props.

### Before

![image](https://user-images.githubusercontent.com/14183168/170231753-18d42757-6b0f-4050-9bca-2f8118c9a0bc.png)

### After

![image](https://user-images.githubusercontent.com/14183168/170231626-e0639fd0-5df8-4143-b1a0-083675ce03f7.png)